pkg/api/api.go

```diff
@@ -89,6 +89,10 @@ func (hs *HTTPServer) registerRoutes() {
 	r.Get("/a/:id/*", reqSignedIn, hs.Index) // App Root Page
 	r.Get("/a/:id", reqSignedIn, hs.Index)
 
+	//pubdash
+	r.Get("/p/:uid/:slug", redirectFromLegacyPanelEditURL, hs.Index)
```

Comment:
these public dashboard routes will probably change. But they're going to be `/p/something` for sure